src/server.js

```diff
@@ -25,6 +25,7 @@ Server.prototype.listen = function(port, host) {
     var client = new Client(true);
     client._end = client.end;
     client.end = function end(endReason) {
+      endReason='{"text":"'+endReason+'"}';
```

Comment:
wiki.vg says "chat" for both 1.8 and 1.9, but 1.8 server sends '"reason"' while 1.9 sends '{"text":"reason"}' which should both be valid in the "chat" format (a quoted string is a json value).

So there are 2 sides to this : 
* the tests should just check that the quoted reason is contained in the message to be cross-version
* we should check sending '{"text":"reason"}' make the vanilla client display the reason properly